runtime/src/did/tests.rs

```diff
@@ -17,8 +17,7 @@
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
 use super::*;
-use primitives::*;
-use primitives::{Blake2Hasher, H256};
+use primitives::{Blake2Hasher, H256, *};
```

Comment:
and here

Reply:
what works here is `use primitives::{ed25519, Blake2Hasher, H256, Pair};` - it does complain about unused imports though (Pair)

Reply:
Strange, I don't get a warning here. 😕